docs/modules/ROOT/pages/reference/service-maturity.adoc

```diff
@@ -59,18 +59,19 @@ See also:
 As a user of the service I can:
 
 * see metrics of a service instance
+* see SLO reports
 * access a dashboard showing pre-crafted diagrams
 
 == Iteration: Alerting
 
 As a user of the service I can:
 
 * enable alerting on pre-defined SLIs to VSHN so that VSHN can resolve incidents with priority
-* see SLO reports so that I can get an overview of how well my instance performs
+* enable alerting on pre-defined incident prevention to me so that I can prevent potential incidents
```

Comment:
I have a hard time to understand this sentence. Would you be able to do an iteration over it?

Reply:
Right, this is unclear.
The idea is that the customer gets alerts like "storage is getting full", not VSHN. 

Would this be clearer? Using the same wording as we have in the product description. 
"enable alerting on the capacity of the instances to an alert channel of my choice so that I can prevent incidents"

Reply:
That sounds much clearer to me, yes.

Reply:
Perfect :+1: